src/V3Config.cpp

```diff
@@ -610,7 +610,7 @@ void V3Config::applyIgnores(FileLine* filelinep) {
 }
 
 void V3Config::applyModule(AstNodeModule* modulep) {
-    const string& modname = modulep->name();
+    const string& modname = modulep->origName();
```

Comment:
Sorry I forgot to explain why adding this. I have added a comment to clarify it.

It's needed because recursive clone modules ("__Vrcm" modules) also need these configs applied.

Reply:
Perhaps I'm looking in the wrong spot but don't see an added comment?

Reply:
I added it here (but it's not a dedicated test for recursive modules) https://github.com/verilator/verilator/pull/4654/commits/1bebd8ec5ef90cdf938f190c2d20405affaabb46